main.go

```diff
@@ -0,0 +1,215 @@
+package main
+
+import (
+	"database/sql"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+	"regexp"
+	"time"
+
+	"github.com/howeyc/fsnotify"
+	_ "github.com/lib/pq"
+)
+
+var (
+	// example: 2014-10-31-15-15.json
+	filenameRegex     = regexp.MustCompile(`(\d{4}(-\d{2}){4})\.json$`)
+	datetimeRegex     = regexp.MustCompile(`([\d-]*)`)
+	datetimeFormat    = "2006-01-02-15-04"
```

Comment:
whoa.... fancy

It just figures it out?  How does it know it's not `Mon Feb 1 15:04:05 MST 2006`?